midend/expandLookahead.h

```diff
@@ -36,6 +36,12 @@ class DoExpandLookahead : public Transform {
     P4::ReferenceMap* refMap;
     P4::TypeMap* typeMap;
     IR::IndexedVector<IR::Declaration> newDecls;
+    /**
```

Comment:
I think you will have to document this flag in the class documentation as well, otherwise people may not notice it.
This is in fact a "policy" flag.

Reply:
done in 3a2ab3e2190678c87b9a8cba35fc181f18c91eea